Tests/Test.Eyes.Selenium.DotNet/TestSetup.cs

```diff
@@ -438,19 +438,19 @@ public void Teardown()
                     }
                     testData.Eyes.Logger.Log("Mismatches: " + results.Mismatches);
                 }
-                if (testData?.Eyes.activeEyes_ is VisualGridEyes visualGridEyes && visualGridEyes.eyesConnector_ is Mock.MockEyesConnector mockEyesConnector)
-                {
-                    RenderRequest[] lastRequests = mockEyesConnector.LastRenderRequests;
-                    string serializedRequests = JsonUtils.Serialize(lastRequests);
-                    if (!TestUtils.RUNS_ON_CI)
-                    {
-                        string dateString = DateTime.Now.ToString("yyyy_MM_dd__HH_mm");
-                        string directory = Path.Combine(TestUtils.LOGS_PATH, "DotNet", "VGResults", dateString);
-                        Directory.CreateDirectory(directory);
-                        File.WriteAllText(Path.Combine(directory, testData.TestNameAsFilename + ".json"), serializedRequests);
-                    }
-                    Assert.AreEqual(testData.ExpectedVGOutput, serializedRequests, "VG Request DOM JSON");
-                }
+                //if (testData?.Eyes.activeEyes_ is VisualGridEyes visualGridEyes && visualGridEyes.eyesConnector_ is Mock.MockEyesConnector mockEyesConnector)
```

Comment:
remove?

Reply:
no, we might some day want to re-enable this code, better sooner than later.
it makes all the tests that have relevant JSON expected results file in UFG run without actually going out for communication.